server/routes/scheduleTracker/controller.js

```diff
@@ -1,14 +1,16 @@
 import pool from "../../db";
+import { getUserId } from "../../utils/getUserId";
 import { objectToQuery } from "../../utils/objectToQuery";
 
 export const getSchedules = (req, res) => {
 	const { searchQuery } = req.query;
 	const query = {
-		text: "SELECT * FROM scheduleTracker",
+		text: "SELECT school.name as school, initiative.name as initiative, scheduleTracker.duration, scheduleTracker.numofnewstudents, scheduleTracker.numofexistingstudents,	scheduleTracker.numofnewteachers, scheduleTracker.numofexistingteachers, scheduleTracker.grades, scheduleTracker.languagestaught, scheduleTracker.totalnumtablets, scheduleTracker.supportcategory,	scheduleTracker.supporttype FROM scheduleTracker inner join school on school.id = scheduleTracker.schoolId inner join initiative on initiative.id = scheduleTracker.programmeInitiativeId",
```

Comment:
Irina could you please make this a string that fits on one page by either
1) text = "...."
text += "..."
or better still
2) put it within backquotes ``
so that you can write 
text = `text: "SELECT school.name as school, initiative.name as initiative, 
scheduleTracker.duration, scheduleTracker.numofnewstudents,
...`;

Thank you

Reply:
Done